codegen/compile.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
According to the latest revision this hasn't actually been changed: https://raw.githubusercontent.com/opesci/propagator/codegen/codegen/compile.py
Were some commits lost in the refactoring of the commit history? The same goes for several other comments - please go through and check.